chart/templates/flower/flower-deployment.yaml

```diff
@@ -18,7 +18,8 @@
 ################################
 ## Airflow Flower Deployment
 #################################
-{{- if eq .Values.executor "CeleryExecutor" }}
+{{- $redis_is_deployed_by_chart := not (or .Values.redis.brokerURLSecretName .Values.redis.brokerURL) }}
```

Comment:
I wouldn't do that, we can deploy flower even if we have a redis (and I think I checked that it worked properly)?

Reply:
I'd need to check that too. Can add an enabled toggle to the flower if you feel that's more appropriate ? (or should we always have a flower?)